test/MUXControlsTestApp/App.xaml

```diff
@@ -35,6 +35,153 @@
                 <Setter Property="BorderThickness" Value="1"/>
                 <Setter Property="Padding" Value="8"/>
             </Style>
+
+            <Style TargetType="local:InfoBar" >
```

Comment:
> <Style TargetType="local:InfoBar" > [](start = 11, length = 36)

This should exist in a separate file which gets included into the app, that file should live in the same C# shared project as the InfoBar definition that gets included as a reference into the test app.

Reply:
Makes sense - I removed it from App.xaml and put it underneath InfoBarTestUI